packages/esm-reports-app/src/components/overview.component.tsx

```diff
@@ -0,0 +1,350 @@
+      });
+    }
+  }, []);
+
+  const processAndDownloadFile = (file) => {
+    const decodedData = window.atob(file.fileContent);
+    const byteArray = new Uint8Array(decodedData.length);
+    for (let i = 0; i < decodedData.length; i++) {
+      byteArray[i] = decodedData.charCodeAt(i);
+    }
+    const url = window.URL.createObjectURL(new Blob([byteArray]));
+    const link = document.createElement('a');
+    link.href = url;
+    link.setAttribute('download', file.filename);
+    document.body.appendChild(link);
+    link.click();
+    document.body.removeChild(link);
+  };
+
+  const clearReportCheckboxes = () => {
+    setCheckedReportUuidsArray([]);
+  };
+
+  return (
+    <div>
+      <ExtensionSlot name="breadcrumbs-slot" className={styles.breadcrumb} />
+      <div className={styles.mainPanelDiv}>
+        <div className={styles.reportsLabelDiv}>
+          <h3>{t('reports', 'Reports')}</h3>
+        </div>
+        <div className={styles.mainActionButtonsDiv}>
+          <Button
+            kind="ghost"
+            renderIcon={() => <Download size={16} style={{ fill: '#0F62FE' }} className={styles.actionButtonIcon} />}
+            iconDescription="Download reports"
+            onClick={() => handleDownloadMultipleReports(checkedReportUuidsArray.join(','))}
+            className={`${styles.mainActionButton} ${
+              downloadReportButtonVisible ? styles.downloadReportsVisible : styles.downloadReportsHidden
+            }`}
+          >
+            {t('downloadReports', 'Download reports')}
+          </Button>
+          <Button
+            kind="ghost"
+            renderIcon={() => <Play size={16} style={{ fill: '#0F62FE' }} className={styles.actionButtonIcon} />}
+            iconDescription="Run reports"
+            onClick={() => {
+              launchOverlay(
+                t('runReport', 'Run Report'),
+                <RunReportForm
+                  closePanel={() => {
+                    closeOverlay();
+                    mutateReports();
+                  }}
+                />,
+              );
+            }}
+            className={styles.mainActionButton}
+          >
+            {t('runReports', 'Run reports')}
+          </Button>
+          <Overlay />
+          <Button
+            kind="ghost"
+            renderIcon={() => <Calendar size={16} style={{ fill: '#0F62FE' }} className={styles.actionButtonIcon} />}
+            iconDescription="Report schedule"
+            onClick={() => navigate({ to: `\${openmrsSpaBase}/reports/scheduled-overview` })}
+            className={styles.mainActionButton}
+          >
+            {t('reportSchedule', 'Report schedule')}
+          </Button>
+        </div>
+      </div>
+      <DataTable rows={reports} headers={tableHeaders} isSortable>
+        {({ rows, headers }) => (
+          <TableContainer>
+            <Table>
+              <TableHead>
+                <TableRow>
+                  <th></th>
```

Comment:
Why do we need the empty `th`?

Reply:
we have table where we have few columns but first column contains checkboxes and this columns has no title, so I put empty th here to keep table aligned